wallet/client.go

```diff
@@ -102,10 +103,13 @@ func (c *grpcClient) getTransaction(id tx.ID) (*pactus.GetTransactionResponse, e
 
 func (c *grpcClient) getFee(amount int64, payloadType payload.Type) (int64, error) {
 	res, err := c.transactionClient.CalculateFee(c.ctx,
-		&pactus.CalculateFeeRequest{Amount: amount, PayloadType: pactus.PayloadType(payloadType)})
+		&pactus.CalculateFeeRequest{
+			Amount:      util.ChangeToCoin(amount),
+			PayloadType: pactus.PayloadType(payloadType),
+		})
 	if err != nil {
 		return 0, err
 	}
 
-	return res.Fee, nil
+	return util.CoinToChange(res.Fee), nil
```

Comment:
Change to Coin! Amir do you tests these APIs?
Please make sure we test them, honestly we don't have chance to fix them after release.
Did you read above thread?

Reply:
@amirvalhalla Apologize for the mistake. here is the client API, so it is correct.